package.json

```diff
@@ -21,12 +21,13 @@
   },
   "dependencies": {
     "axios": "0.27.2",
+    "nanoid": "3.3.4",
     "react": "18.1.0",
     "react-dom": "18.1.0",
     "react-helmet": "6.1.0",
+    "react-icons": "^4.4.0",
```

Comment:
Are you really using the react-icons library? 🤔

Reply:
Not yet, later on we want to use it maybe. We tried it at first but first KISS :D